src/content/docs/de/tutorial/3-components/1.mdx

```diff
@@ -0,0 +1,156 @@
+</PreCheck>
+
+## Erstelle einen neuen `src/components/`-Ordner
+
+Um `.astro`-Dateien aufzunehmen, die HTML generieren, aber keine neuen Seiten auf deiner Website werden, benötigst du einen neuen Ordner in deinem Projekt: `src/components/`.
+
+
+## Erstelle eine Navigationskomponente
+
+<Steps>
+1. Erstelle eine neue Datei: `src/components/Navigation.astro`.
+
+2. Kopiere deine Links für die Navigation zwischen den Seiten aus dem oberen Bereich einer beliebigen Seite und füge sie in deine neue Datei `Navigation.astro` ein:
+
+    ```astro title="src/components/Navigation.astro" "---"
+    ---
+    ---
+    <a href="/">Home</a>
+    <a href="/about/">Über mich</a>
+    <a href="/blog/">Blog</a>
+    ```
+    :::tip
+    Wenn im Frontmatter deiner `.astro-Datei` nichts steht, musst du auch die Code-Abgrenzung nicht schreiben. Du kannst sie jederzeit wieder hinzufügen, wenn du sie brauchst.
+    :::
+</Steps>
+
+### Importieren und Verwenden von Navigation.astro
+
+<Steps>
+1. Gehe zurück zu `index.astro` und importiere deine neue Komponente innerhalb der Code-Abgrenzung:
+
+    ```astro title="src/pages/index.astro"
+    ---
+    import Navigation from '../components/Navigation.astro';
+    ---
+    ```
+
+2. Ersetze dann darunter die vorhandenen Navigations-HTML-Link-Elemente durch die neu importierte Navigationskomponente:
+
+    ```astro title="src/pages/index.astro" del={1-3} ins={4}
+    <a href="/">Home</a>
+    <a href="/about/">Über mich</a>
+    <a href="/blog/">Blog</a>
+    <Navigation />
+    ```
+
+3. Überprüfe die Vorschau in deinem Browser und stelle fest, dass sie genau gleich aussehen sollte... und das ist genau das, was du erreichen wolltest!
+</Steps>
+
+Deine Website enthält nun denselben HTML-Code wie zuvor. Aber jetzt werden diese drei Zeilen Code durch deine `<Navigation />`-Komponente bereitgestellt.
+
+<Box icon="puzzle-piece">
+
+## Probiere es selbst aus - Füge Navigation zum Rest deiner Website hinzu
+
+Importiere die `<Navigation />`-Komponente und verwende sie auf den anderen beiden Seiten deiner Website (`about.astro` und `blog.astro`) mit derselben Methode.
+
+Vergiss nicht,
+- eine Import-Anweisung am Anfang des Komponenten-Skripts, innerhalb der Code-Abgrenzung, hinzuzufügen.
+- den vorhandenen Code durch die Navigationskomponente zu ersetzen.
+
+</Box>
+
+:::note
+Wenn du deinen Code umstrukturierst, aber das Aussehen deiner Seite im Browser nicht veränderst, **refaktorisierst** du. In dieser Einheit wirst du mehrmals **refaktorisieren**, indem du Teile deines Seiten-HTMLs durch Komponenten ersetzt.
+
+Das ermöglicht es dir, schnell mit beliebigem funktionierenden Code zu beginnen, der oft in deinem Projekt dupliziert wird. Dann kannst du das Design deines bestehenden Codes schrittweise verbessern, ohne das äußere Erscheinungsbild deiner Website zu verändern.
+:::
+
+
+
+<Box icon="question-mark">
+
+### Test your knowledge
+
+
+
+Das kannst du tun, wenn Elemente auf mehreren Seiten wiederholt werden:
```

Comment:
```suggestion
1. Das kannst du tun, wenn Elemente auf mehreren Seiten wiederholt werden:
```
Zumindest finde ich in der englischen Version einen Einser davor 🙂

Reply:
Whoops muss verloren gegangen sein ^^'